src/pages/venues/[id]/map/map.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "breakpoints";
+@import "variables";
+
+.content {
+  --background-color: #{$color-engel-medium-light};
+  grid-column: 1 / -1 !important;
+}
+
+.title {
+  font-size: 24px;
+  margin-left: var(--spacing-xl);
```

Comment:
We've used SASS variables because it allows us to lock HDS design tokens into a specific version. Because CSS variables are global, they can be altered by dependencies. Downside is that you lose themability interop.

Not a major point of concern for me, FYI.